<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toOracleSettingUI</class>
 <widget class="QWidget" name="toOracleSettingUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="4" column="1">
    <widget class="QPushButton" name="CreatePlanTable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Create Table</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="DefaultDate"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="TextLabel6_2_2_3">
     <property name="toolTip">
      <string>The default timestamp format to use when querying the database.</string>
     </property>
     <property name="text">
      <string>Default &amp;timestamp format</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>DefaultTimestamp</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="UnlimitedBool">
     <property name="text">
      <string>Unlimited</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="VsqlPlansBool">
     <property name="toolTip">
      <string>Use V$SQL_PLAN when possible.</string>
     </property>
     <property name="text">
      <string>use &amp;V$SQL_PLAN</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="TextLabel6_3">
     <property name="toolTip">
      <string>Name of the table to put explain plan output in.</string>
     </property>
     <property name="text">
      <string>&amp;Explain plan table</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>ExplainPlan</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="SharedPlanBool">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>shared PLAN_TABLE</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="KeepPlansBool">
     <property name="toolTip">
      <string>Keep records for the plan explanations in the plan table.</string>
     </property>
     <property name="text">
      <string>&amp;Keep plans</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="ExplainPlan"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="MaxLongInt"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="TextLabel6_3_2">
     <property name="toolTip">
      <string>Max length to read from LOB or LONG data fields. LONG:s can not be specified unlimited but will be defaulted to 33000 if unlimited.</string>
     </property>
     <property name="text">
      <string>&amp;Max long and LOB data length</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>ExplainPlan</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QGroupBox" name="extractorGroupBox">
     <property name="title">
      <string>Extractor options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0">
      <item>
       <widget class="QCheckBox" name="UseDbmsMetadataBool">
        <property name="text">
         <string>Use DBMS_METADATA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="IncludeStorageBool">
        <property name="text">
         <string>Include storage information</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SkipOrgMonBool">
        <property name="text">
         <string>     Skip organisation and monitoring</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SkiptStorExTablespaceBool">
        <property name="text">
         <string>     Skip storage except tablespace</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="IncludeParallelBool">
        <property name="text">
         <string>Include parallel information</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="IncludePartitionBool">
        <property name="text">
         <string>Include partition information</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="IncludeCodeBool">
        <property name="text">
         <string>Include code</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="IncludeHeaderBool">
        <property name="text">
         <string>Include header</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="IncludePromptBool">
        <property name="text">
         <string>Include prompt</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="TextLabel6_2_2_2">
     <property name="toolTip">
      <string>The default dateformat to use when querying the database.</string>
     </property>
     <property name="text">
      <string>&amp;Default date format</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>DefaultDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="DefaultTimestamp"/>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>DefaultDate</tabstop>
  <tabstop>DefaultTimestamp</tabstop>
  <tabstop>MaxLongInt</tabstop>
  <tabstop>UnlimitedBool</tabstop>
  <tabstop>ExplainPlan</tabstop>
  <tabstop>KeepPlansBool</tabstop>
  <tabstop>CreatePlanTable</tabstop>
  <tabstop>VsqlPlansBool</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>CreatePlanTable</sender>
   <signal>clicked()</signal>
   <receiver>toOracleSettingUI</receiver>
   <slot>createPlanTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UnlimitedBool</sender>
   <signal>toggled(bool)</signal>
   <receiver>MaxLongInt</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1()</signal>
 </slots>
</ui>
